FirstAPP/src/Main.kt

```diff
@@ -0,0 +1,4 @@
+fun main(args : Array<String>){
```

Comment:
Your project source files should be in `src/main/kotlin` and you already have on there. Hence this one is not needed.